src/test/java/DeadlineTest.java

```diff
@@ -0,0 +1,17 @@
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+import duke.Deadline;
+
+/**
+ * A class to test the methods in <code>Deadline</code>.
+ */
+public class DeadlineTest {
+
+    @Test
+    public void testDeadlineToString() {
```

Comment:
Maybe give it a better name to include what is the input and what is the expected behaviour?